presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -144,7 +144,8 @@ public void shouldRunBatchQuery()
         assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
     }
 
-    @Test(groups = CLI, timeOut = TIMEOUT)
+    //@Test(groups = CLI, timeOut = TIMEOUT)
```

Comment:
I want to run `CLI` group explicitly. So selecting `cli` group with `-g cli` will pick the tests from the `quarantine` group as well.